nixos/modules/services/misc/overseerr.nix

```diff
@@ -0,0 +1,62 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.services.overseerr;
+in
+{
+  meta.maintainers = [ ];
+
+  options.services.overseerr = {
+    enable = mkEnableOption (mdDoc ''Overseerr, a requests manager for Plex'');
+
+    openFirewall = mkOption {
+      type = types.bool;
+      default = false;
+      description = mdDoc ''Open port in the firewall for the Overseerr web interface.'';
+    };
+
+    port = mkOption {
+      type = types.port;
+      default = 5055;
+      description = mdDoc ''The port which the Overseerr web UI should listen to.'';
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.services.overseerr = {
+      description = "Overseerr, a requests manager for Plex";
+      after = [ "network.target" ];
+      wantedBy = [ "multi-user.target" ];
+      environment.PORT = toString cfg.port;
+      serviceConfig = {
+        Type = "exec";
+        StateDirectory = "overseerr";
+        WorkingDirectory = "${pkgs.overseerr}/libexec/overseerr/deps/overseerr";
+        DynamicUser = true;
+        ExecStart = "${pkgs.overseerr}/bin/overseerr";
+        BindPaths = [ "/var/lib/overseerr/:${pkgs.overseerr}/libexec/overseerr/deps/overseerr/config/" ];
```

Comment:
Not a blocker, but I see the package is being referred to a lot in this setup. Consider adding a `package` option to make it easier for users to override the contents of these directories.